### PR TITLE
Organizes the controller folder

### PR DESCRIPTION
**Pull request by kd7uiy**
The Controller folder is pretty complex, and this change should help to make that change a bit easier to find things

### CODE REVIEW

File: Assets/Scripts/Controllers/UIController.cs [hunk 1]
```diff
@@ -1,23 +0,0 @@
-﻿#region License
```
Comment (mikejbrown) [inline Assets/Scripts/Controllers/UIController.cs]: Question: this class was apparently added for future extensibility, but so far does nothing. Should it be removed now? Here are the commits which touch it:
```
$ git slog -- Assets/Scripts/Controllers/UIController.cs
* [2016-09-16] 6940a61  Organizes the controller folder (Ben Pearson)
* [2016-08-27] 8257d1e  UIController.cs SCS (#893) (vringar)
* [2016-08-22] 73b8d8b  adds headers (#600) (Zak)
* [2016-08-20] 7457090  UI Controller (#420) (Jack Evans)
```

The full log on the first one is
```
commit 7457090957d4923cf743751e898f5a37adf1eb6b
Author: Jack Evans <j-j-evans@hotmail.com>
Date:   Sat Aug 20 18:25:27 2016 +0100

    UI Controller (#420)

    * Created UI Controller for future code driven UI.

    * Instantiated UI Controller

    UI controller now instantiated by WorldController to fit with
    https://github.com/TeamPorcupine/ProjectPorcupine/issues/403
```

Comment (mikejbrown) [inline Assets/Scripts/Controllers/UIController.cs]: @TheAkashicTraveller

Comment (kd7uiy) [inline Assets/Scripts/Controllers/UIController.cs]: My thought is, let's add it back when it's actually being used, but for right now, I'm not convinced it's needed.

Comment (sboigelot) [inline Assets/Scripts/Controllers/UIController.cs]: agree with @kd7uiy

**Comment by mikejbrown** [comment]
This pull request has merge conflicts. Please resolve the conflicts and remove [Conflicts] from the title.

**Comment by mikejbrown** [comment]
Going with the decisions to delete UIController for now. It can always be restored if needed.

Now :pray: :lipstick: :goat: about all the new merge conflicts.

**Comment by TomMalbran** [comment]
Several classes in Controller are not even Controllers. Why do we keep leaving them there?

"Input Output" shouldn't have spaces

**Comment by mikejbrown** [comment]
@TomMalbran This PR has already been merged. Please open a new PR or Issue with your proposed changes.

**Comment by TomMalbran** [comment]
I know. It was for someone to read and make the change or discuss about it. But I will change it.

**Pull request merged**